chaco/shell/commands.py

```diff
@@ -655,7 +655,7 @@ def yaxis(**kwds):
     p = curplot()
     if p:
         if kwds:
-            p.y_axis.set(**kwds)
+            p.y_axis.trait_setq(**kwds)
```

Comment:
`trait_set` here?  I guess we can assume none of the keys in `kwds` is `"trait_change_notify"`?

Reply:
You're right, that's incorrect. I've made this into a call to `.trait_set`.